rel/runtime_config.exs

```diff
@@ -0,0 +1,15 @@
+use Mix.Config
+
+[
+  "PRODUCTION_ENVIRONMENT"
```

Comment:
I guess IDCTA will come at a later PR isn't it?
https://github.com/bbc/belfrage/blob/master/rel/runtime_config.exs#L25

Reply:
Fetcher doesn't use IDCTA.